fastlane/lib/fastlane/actions/cocoapods.rb

```diff
@@ -15,6 +15,7 @@ def self.run(params)
         end
 
         cmd << ['bundle exec'] if File.exist?('Gemfile') && params[:use_bundle_exec]
+        cmd << ['pod update &&'] if params[:use_pod_update]
```

Comment:
When both `use_pod_update` and `use_bundle_exec` are set, this will cause only pod update to be run using bundle exec, because the whole command will be:

```
bundle exec pod update && pod install
```